index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <link rel="stylesheet" type="text/css" href="resume.css">
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+      <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;1,100;1,200&display=swap" rel="stylesheet">
+      <title>Resume</title>
+   </head>
+   <body>
+      <div class="header">
+         <h1>CHALISA KASETSAWAD</h1>
+         <h2>Freelancer</h2>
+      </div>
```

Comment:
Better to use `header` tag for better meaning of the code and SEO
```suggestion
      <headerclass="header">
         <h1>CHALISA KASETSAWAD</h1>
         <h2>Freelancer</h2>
      </header>
```

ref: https://www.w3schools.com/tags/tag_header.asp